custom-service-adapter/pom.xml

```diff
@@ -0,0 +1,132 @@
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>knotx.io.tutorial</groupId>
+  <artifactId>custom-service-adapter</artifactId>
+  <version>1.0.0</version>
+  <name>Custom service adapter</name>
+
+  <properties>
+    <!-- the main verticle class name -->
+    <main.verticle>com.cognifide.knotx.launcher.KnotxStarterVerticle</main.verticle>
+    <file.encoding>UTF-8</file.encoding>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>1.8</java.version>
+    <handlebars.version>4.0.5</handlebars.version>
+    <vertx.version>3.3.3</vertx.version>
+    <rxjava.version>1.1.9</rxjava.version>
+    <knotx.version>1.0.0</knotx.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.knotx</groupId>
+      <artifactId>knotx-core</artifactId>
+      <version>${knotx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.knotx</groupId>
+      <artifactId>knotx-adapter-common</artifactId>
+      <version>${knotx.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.1.3</version>
+    </dependency>
+
+    <!-- CORE -->
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-core</artifactId>
+      <version>${vertx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java</artifactId>
+      <version>${vertx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-jdbc-client</artifactId>
+      <version>${vertx.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.hsqldb</groupId>
+      <artifactId>hsqldb</artifactId>
+      <version>2.3.4</version>
+    </dependency>
+    <dependency>
+      <groupId>io.reactivex</groupId>
+      <artifactId>rxjava</artifactId>
+      <version>${rxjava.version}</version>
+    </dependency>
+
+    <!-- COMMON -->
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.3.2</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.1</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.0.2</version>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
```

Comment:
Setting this up and explaining the purpose of the plugin could makre for an interesting lightning talk/blog post.